docs/developers/api.md

```diff
@@ -9,102 +8,35 @@ Example of these apps include:
-
-## Transaction Endpoints
-### POST /transactions-pool
-Send L2 transaction. The transaction will be stored in the transaction pool of the coordinator and eventually forged.
-
-### GET /transactions-pool/{id}
-Get transaction from the pool by its id. This endpoint is specially useful for tracking the status of a transaction that may not be forged yet.
-Only transactions from the pool will be returned.
-Note that the transaction pool is different for each coordinator and therefore only a coordinator that has received a specific transaction
-will be able to provide information about that transaction.
-
-### GET /transactions-history
-Get historical transactions. This endpoint will return all the different types of transactions except for:
-- Transactions that are still in the transaction pool of any coordinator. These transactions can be fetched using `GET /transactions-pool/{id}`.
-- L1 transactions sent by users that have not been forged yet. These transactions can be fetched using `GET /transactions-history/{id}`.
-
-### GET /transactions-history/{id}
-Get transaction from the history by its id. This endpoint will return all the different types of transactions except those that are still in the pool of any coordinator.
-
-## Hermez Status Endpoints
-### GET /batches/{batchNum}
-Get information on a specific batch.
-
-### GET /full-batches/{batchNum}
-Get information on a specific batch, including the associated transactions. The object returned in this method can be a bit heavy. 
-If you're developing a front end, you may consider using a combination of `GET /batches/{batchnum}` and `GET /history-transactions?batchNum={batchNum}`.
-
-### GET /slots
-Get information about slots.
-
-### GET /slots/{slotNum}
-Get information about a specific slot.
-
-### GET /bids
-Get a list of bids made for a specific slot auction.
-
-### GET /state
-Return information that represents the current state of the network. It also includes metrics and statistics.
-
-### GET /config
-Return constant configuration of the network.
 
-### GET /tokens
-Get information of the supported tokens in the Hermez network.
+## Account
+This group of endpoints provides `account` and `exits` related information.
 
-### GET /tokens/{id}
-Get information of a token supported by Hermez network.
+There are two sets of `account` operations that can be performed:
+- Retrieving information from existing accounts, using `/accounts` endpoints.
+- Retrieving/configuring account authorizations using `/account-creation-authorization` endpoints. It is possible that a user authorizes the Coordinator the creation of an account with an Ethereum address using `POST /account-creation-authorization` endpoint. Then, the Coodinator will create the account at the moment this account receives a `Transfer`.
```

Comment:
"...that a user authorizes the Coordinator to create an account with an Ethereum address..."